diff --git a/node_modules/d3-sankey/src/sankey.js b/node_modules/d3-sankey/src/sankey.js
index 56ad642..a593c86 100644
--- a/node_modules/d3-sankey/src/sankey.js
+++ b/node_modules/d3-sankey/src/sankey.js
@@ -190,12 +190,16 @@ export default function Sankey() {
     }
   }
 
+  const alignReplaced = function r(n, t) {
+    return n.sourceLinks.length ? n.depth : t - 1;
+  }
+
   function computeNodeLayers({nodes}) {
     const x = max(nodes, d => d.depth) + 1;
     const kx = (x1 - x0 - dx) / (x - 1);
     const columns = new Array(x);
     for (const node of nodes) {
-      const i = Math.max(0, Math.min(x - 1, Math.floor(align.call(null, node, x))));
+      const i = Math.max(0, Math.min(x - 1, Math.floor(alignReplaced(node, x))));
       node.layer = i;
       node.x0 = x0 + i * kx;
       node.x1 = node.x0 + dx;
